src/main/java/uk/gov/pay/api/model/CardDetails.java

```diff
@@ -73,4 +76,9 @@ public String getExpiryDate() {
     public String getCardBrand() {
         return cardBrand;
     }
+
+    @ApiModelProperty(value = "The card type, `debit` or `credit` or `null` if not able to determine", allowableValues = "debit,credit,null", example = "debit")
```

Comment:
how about `...or null if unknown`?

Reply:
"if not able to determine" was what Mark and Alex decided on

Reply:
The use of “not able to determine” comes from conversations with @m-green and will match the phrasing in the documentation.

Reply:
Just a heads-up that I'm likely to tweak this wording slightly, sorry!

Reply:
But I wouldn't let that get in the way of this PR.

Reply:
👍